### ISSUE TITLE
Unable to install 0.93 on python2 machines

### ISSUE THREAD

**Issue by tedivm** [ParallelSSH/parallel-ssh#70]
I'm getting this error when attempting to install the newest version of the module. I have not had this issue on previous versions.

```
Collecting parallel-ssh (from pyvcluster->-r /home/jenkins-slave/workspace/####/requirements.txt (line 20))
  Downloading parallel-ssh-0.93.0.tar.gz
    Complete output from command python setup.py egg_info:
    error in parallel-ssh setup command: Invalid environment marker: python_version < "3"
```

**Comment by pkittenis** [comment]
Can you please run `pip install -U pip` and try again? `0.93.0` contains python version dependent requirements which needs newer version of pip than earlier releases.

**Comment by pgigis** [comment]
I had the same issue and "pip install -U pip" solve it.

**Comment by pkittenis** [comment]
Thanks for confirming @pgigis, will add a pip dependency requirement on the readme.

The other error above is from installation of `cryptography`, a dependency of `paramiko`, which requires openssl development headers package to build its native extensions (``openssl/opensslv.h: No such file or directory``).  These are in `libssl-dev` package on debian derived systems. Error is not related to `parallel-ssh`.

The `pyopenssl` package otoh provides bindings to the openssl libraries as pure python code and is a (slower) alternative to having to install openssl development headers.

In short, either install `libssl-dev` or equivalent package so that cryptography can build its native extensions or use `pyopenssl`. Cryptography by default will try to build native extensions against openssl development headers.

Note that cryptography with native extensions without pyopenssl is likely to be faster and with fewer python package dependencies.

From my system with `libssl-dev` installed installing cryptography pulls in:

```
Successfully installed cffi-1.9.1 cryptography-1.7.1 enum34-1.1.6 idna-2.2 ipaddress-1.0.17 pyasn1-0.1.9 pycparser-2.17 six-1.10.0
```
